examples/cli-version-compare/connector-required:algo+postgres#images-not-present/o.txt

```diff
@@ -0,0 +1,12 @@
+New images are available for:
+ * reachsh/devnet-algo   8e78dffa:  stable, 0, 0.1, 0.1.8, 34107655, 2022-01-31
```

Comment:
One thing about these that I was noticing... this is telling you want is NEW but it doesn't show at all what you HAVE... I think that may be interesting... because you want to know just how old the thing you're on is

Reply:
I've created CORE-1169 for this.